docs/authentication/saml/azure.mdx

```diff
@@ -101,16 +107,35 @@ description: Learn how to allow users to sign into your Clerk app with their Mic
 
   The only Microsoft claim that is necessary to map is the email address claim. This is the email address that your users will use to log in to your application.
 
+  1. Navigate back to the **Microsoft Azure portal**.
   1. On the **Set up Single Sign-On with SAML** page, find the **Attributes & Claims** section.
   1. Select the **Edit** button.
-  1. Select the email address claim (`http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress`) to edit the field.
-  1. Next to **Source attribute**, select the dropdown and choose `user.userprincipalname`.
+  1. To edit the email claim, select the claim with the value of **`user.email`**. You will be redirected to the **Manage claim** page.
+  1. Next to **Source attribute**, search and select the dropdown and choose **`user.userprincipalname`**.
   1. Select **Save** at the top of the page.
 
   ### Enable the connection for Clerk
 
   To make the connection available for your users to authenticate with:
 
-  1. In the Clerk Dashboard, you should still have the connection's configuration page open. If not, navigate to the [**SSO Connections**](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections) page and select the connection.
-  1. Toggle on **Enable connection** and select **Save**.
+  1. Navigate back to the **Clerk Dashboard** to the [**SSO Connections**](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections) page.
+  1. Under **Connection**, toggle on **Enable connection** and select **Save**.
```

Comment:
they never shy away from the "Connection" tab, and its also the default tab, so that part is a given and isn't necessary to mention.

```suggestion
  1. At the top of the page, toggle on **Enable connection** and select **Save**.
```